app/MMB/src/scenes/tickets/boardingPasses/boardingPassInformation/BoardingPassInformation.js

```diff
@@ -14,27 +13,24 @@ type PropsWithContext = {|
   +isPastBooking: boolean,
 |};
 
-const BoardingPassInformation = (props: PropsWithContext) => {
-  const availableAt = idx(props.data, _ => _.availableAt) || '';
+const BoardingPassInformation = ({ data, isPastBooking }: PropsWithContext) => {
+  const availableAt = data.availableAt ?? '';
   const [year, month, day] = availableAt.split('-');
   const date = availableAt
     ? new Date(Date.UTC(parseInt(year), parseInt(month) - 1, parseInt(day)))
     : null;
 
-  if (props.isPastBooking) {
+  if (isPastBooking) {
     return (
       <PastBookingInformation
         boardingPassAvailableDate={date}
-        boardingPassUrl={idx(props.data, _ => _.boardingPassUrl)}
+        boardingPassUrl={data.boardingPassUrl}
       />
     );
   }
 
   return (
-    <FutureBookingInformation
-      boardingPassAvailableDate={date}
-      data={props.data}
-    />
+    <FutureBookingInformation boardingPassAvailableDate={date} data={data} />
```

Comment:
This is one of the cases where the variable names are too close to each other that it could be confusing if reading too fast :D

Reply:
renamed the variable 👍